peer.go

```diff
@@ -416,6 +417,7 @@ func (p *peer) loadActiveChannels(chans []*channeldb.OpenChannel) error {
 		)
 		if err != nil {
 			lnChan.Stop()
+			peerLog.Debugf("unable to add link %v: %v", chanPoint, err)
```

Comment:
Nit: capitalize first word for consistency.

Reply:
fixed! also made peer disconnection logged to info, which seems useful as per discussion in #1630